spec/models/user_spec.rb

```diff
@@ -665,32 +665,6 @@
     end
   end
 
-  describe "can self-delete registration" do
```

Comment:
(You may be able to salvage these tests if you decide to accept my other suggestion above)

Reply:
Kindly ping re salvaging tests. These would need to be ported to `registration_spec` accordingly, but with the trouble of `enums` returning raw ints vs returning the "nice-ified" literal value, and the open question of whether that literal is a string or a sym, having tests won't hurt.

Reply:
The tests are covered in registration_checker_spec - the intention is to move them into registration_spec eventually as part of a larger refactor of the checker, but for now all the test cases I'm concerned about are covered